### ISSUE TITLE
Please add icon to the dockable tab [enhancement]

### ISSUE THREAD

**Issue by vinsworldcom** [sunjw/jstoolnpp#119]
Please add an icon to the dockable tab; otherwise the text alignment does not look good:

Current implementation without icon:
![image](https://user-images.githubusercontent.com/9657169/156805932-262b9d67-28be-49e7-addc-8fffe8bf401a.png)

With icon:
![image](https://user-images.githubusercontent.com/9657169/156805196-6048f1dc-91cf-41c8-829b-2d5abf240a75.png)

It is implemented simply with this patch.  I can submit a pull request if you prefer, but this should be pretty easy to incorporate yourself:

```Diff
diff --git a/trunk/src/jsMinNpp.cpp b/trunk/src/jsMinNpp.cpp
index ec3ffc2..02db6fd 100644
--- a/trunk/src/jsMinNpp.cpp
+++ b/trunk/src/jsMinNpp.cpp
@@ -652,8 +652,10 @@ void jsonTree()
                s_jsonDialog.create(&data);

                // define the default docking behaviour
-               data.uMask = DWS_DF_CONT_LEFT;
+               data.uMask = DWS_DF_CONT_LEFT | DWS_ICONTAB;
+               data.hIconTab = ( HICON )::LoadImage( s_jsonDialog.getHinst(), MAKEINTRESOURCE( IDI_JS_ICON ), IMAGE_ICON, 16, 16, LR_LOADTRANSPARENT );
                data.pszModuleName = s_jsonDialog.getPluginFileName();
                data.pszName = TEXT("JSToolNpp JSON Viewer");
```

Cheers.

**Comment by sunjw** [comment]
Thanks very much, I'll merge this.

**Comment by sunjw** [comment]
Merged in 5920981ca9fcb5cf09d03788cdcc9fbb6a4c563f
